Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.h

```diff
@@ -21,22 +21,21 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface VOKFetchedResultsDataSource : NSObject <NSFetchedResultsControllerDelegate, UITableViewDelegate, UITableViewDataSource>
 
-@property (readonly) Class managedObjectClass;
-@property (weak, readonly) UITableView *tableView;
-@property (weak, readonly) NSManagedObjectContext *managedObjectContext;
+@property (nonatomic, assign, readonly) Class managedObjectClass;
+@property (nonatomic, weak, readonly) UITableView *tableView;
 
-@property (weak) id<VOKFetchedResultsDataSourceDelegate> delegate;
+@property (nonatomic, weak) id<VOKFetchedResultsDataSourceDelegate> delegate;
 
-@property (strong, readonly) NSFetchedResultsController *fetchedResultsController;
+@property (nonatomic, strong, readonly) NSFetchedResultsController *fetchedResultsController;
```

Comment:
1. I don't see a question.
1. Otherwise, how would you set a readonly property (that wasn't computed)?
1. _setter_ is synthesized too?